README-for-review.md

```diff
@@ -0,0 +1,384 @@
+![Logo](https://raw.githubusercontent.com/inaka/Jayme/master/Assets/logo.png)
+
+***The abstraction layer that eases RESTful interconnections in Swift***
+
+------
+
+[![Build Status](https://api.travis-ci.org/inaka/Jayme.svg)](https://travis-ci.org/inaka/Jayme) [![Codecov](https://codecov.io/gh/inaka/jayme/branch/master/graph/badge.svg)](https://codecov.io/gh/inaka/jayme) [![Cocoapods](https://img.shields.io/cocoapods/v/Jayme.svg)](http://cocoadocs.org/docsets/Jayme) [![Twitter](https://img.shields.io/badge/twitter-@inaka-blue.svg?style=flat)](http://twitter.com/inaka)
+
+
+## Overview
+
+What's the best place to put your *entities business logic* code? What's the best place to put your *networking* code?
+
+Jayme answers those two existencial questions by defining a straightforward and extendable architecture based on **Repositories** and **Backends**.
+
+It provides a neat API for dealing with REST communication, leaving your `ViewControllers` out of that business by abstracting all that logic, thereby allowing them to focus on what they should do rather on how they should connect to services.
+
+![Jayme's Architecture In A Nutshell](https://raw.githubusercontent.com/inaka/Jayme/master/Assets/architecture-diagram-1.png)
+
+## Migration Guides
+
+- [Jayme 2.0 Migration Guide](https://github.com/inaka/Jayme/blob/master/Documentation/Jayme%202.0%20Migration%20Guide.md)
+
+##Features
+
+- **Protocol-Oriented**
+  - Jayme has been built following the concepts of [protocol-oriented programming](https://developer.apple.com/videos/play/wwdc2015/408/), encouraging composition over inheritance whenever possible.
+- **Generics**
+  - In order to provide high flexibility, generics and associated types are present almost everywhere in the library.
+- **Error Handling**
+  - Jayme comes with a default list of discrete errors, which are defined in an enumeration named `JaymeError`.
+  - Any `case` in `JaymeError` turns out to be useful from the `ViewController` layer point of view. Cases represent possible different and meaningful UI scenarios. For instance: a `5xx` server status code is represented by a `case ServerError(statusCode: Int)`.
+  - If you need different error definitions, Jayme allows you to use your own Error types, with the aid of associated types.
+- **Futures / Results**
+  - From experience, we've found out that the [Future Pattern](https://realm.io/news/swift-summit-javier-soto-futures/) is a very convenient way for layouting asynchronous code. In consequence, we decided to develop Jayme around that pattern.
```

Comment:
I look up in different dictionaries the word 'layout' used as a verb and I couldn't find it. Let's discuss what do you want to say here and change it for a valid construction.